DownFall/GameViewController.swift

```diff
@@ -13,18 +13,25 @@ import GameplayKit
 class GameViewController: UIViewController {
 
     private var gameSceneNode: GameScene?
+    private var tutorialSceneNode: TutorialScene?
     private var boardSize = 8
     private var entities: [EntityModel]?
-    private var selectedDifficulty: Difficulty = .normal
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        //TODO: handle failure more gracefully. ie redownload or retry
-        let entityData = try! Data.data(from: "entities")!
-        entities = try! JSONDecoder().decode(EntitiesModel.self, from: entityData).entities
-        visitStore(entities![0])
+        do {
+            guard let entityData = try Data.data(from: "entities") else { fatalError("Crashing here is okay because we failed to parse our entity json file") }
+            entities = try JSONDecoder().decode(EntitiesModel.self, from: entityData).entities
+            visitStore(entities![0])
+        }
+        catch(let error) {
+            fatalError("Crashing due to \(error) while trying to parse json entity file")
+        }
+        
+        //MARK: Set default difficulty
+        GameScope.shared.difficulty = .tutorial1
```

Comment:
It does make sense to statically set this here.  In our next bit of work, we need to coordinate a few dependencies to determine our starting difficulty.

Reply:
This will be handled in future tutorial work